plane/source/docs/reverse-thrust-setup.rst

```diff
@@ -0,0 +1,71 @@
+.. _reverse-thrust-setup:
+
+
+====================
+Reverse Thrust Setup
+====================
+
+Reverse thrust can be used in Plane for any auto-throttle mode and in automatic landings, and/or under pilot control to provide steeper descents without gaining airspeed.
+
+In order to use reverse thrust, the ESC must be capable of it. Usually, the ESC input to output will be symmetric with equal forward and reverse output versus input around its input center point.
+
+For the rest of this document, the typical setup of having RC channel 3 control the throttle, and the motor esc being on output 3, for the examples. For multiple motors, the parameters would be used for those output numbers also.
+
+Setup
+=====
+
```

Comment:
We need a giant "REMOVE PROPOLLER BEFORE YOU DO ANYTHING" warning somewhere. This is probably a good spot.